frontends/web/src/api/account.ts

```diff
@@ -267,12 +267,24 @@ export interface UTXOAmount {
     unit: Coin;
 }
 
-export interface UTXO {
+interface UTXO {
     outPoint: string;
     address: string;
     amount: UTXOAmount;
 }
 
-export const getUTXOs = (code: AccountCode): Promise<UTXO[]> => {
-    return apiGet(`account/${code}/utxos`);
+export interface UTXOwithTX extends UTXO {
+    txId: string;
+    txOutput: string;
+}
+
+export const getUTXOs = async (code: AccountCode): Promise<UTXOwithTX[]> => {
+    const utxos: UTXO[] = await apiGet(`account/${code}/utxos`);
+    return utxos.map(utxo => {
+        const [txId, txOutput] = utxo.outPoint.split(':');
```

Comment:
The backend could just pass `outPoint: [txId, txOutput]`, or is there any reason not to?

Reply:
No reason. I did it now in the third commit.